addons/l10n_eg_edi_eta/models/res_company.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+
+from odoo import models, fields
+
+
+class ResCompany(models.Model):
+    _inherit = 'res.company'
+
+    l10n_eg_client_identifier = fields.Char('ETA Client ID', groups="base.group_system")
+    l10n_eg_client_secret_1 = fields.Char('ETA Secret', groups="base.group_system")
```

Comment:
always makes me wonder whether there's a second secret? worth a comment?

Reply:
@odony There is indeed two secrets, I couldn't find any docs explaining the use. My guess is that they offer to secrets in case one was lost (i've seen this done before)
however, both can be used interchangeably, I changed the field name to l10n_eg_client_secret and add a help text